docs/contributing/coding_guide.rst

```diff
@@ -658,16 +658,17 @@ Dependencies and requirements
   - Support for core package dependencies be dropped **2 years** after
     their initial release.
 
-* The |ci_requirements/|_ directory contains pinned requirements files
-  for use in continuous integration tests (see
-  |ci_requirements/README.md|_).
+* The |uv.lock|_ file contains pinned requirements files
+  for use in continuous integration tests.
 
   - These files are updated periodically via pull requests created by a
     GitHub workflow to `update pinned requirements`_.
```

Comment:
Note:  This link currently takes you to the GitHub Action page (https://github.com/PlasmaPy/PlasmaPy/actions/workflows/update-pinned-reqs.yml) instead of the actual workflow script (https://github.com/PlasmaPy/PlasmaPy/blob/main/.github/workflows/update-pinned-reqs.yml).

[Opinion] To me it makes more sense to link to the script and not the Action.  In any case this will not affect this PR.